public/app/features/alerting/unified/NotificationPolicies.tsx

```diff
@@ -67,34 +80,45 @@ const AmRoutes = () => {
 
   const { selectedAlertmanager, hasConfigurationAPI, isGrafanaAlertmanager } = useAlertmanager();
   const { getRouteGroupsMap } = useRouteGroupsMatcher();
-  const { data: muteTimings = [] } = useMuteTimings({ alertmanager: selectedAlertmanager ?? '' });
+  const { data: muteTimings = [] } = useMuteTimings({
+    alertmanager: selectedAlertmanager ?? '',
+    skip: !showTimingsTab,
+  });
 
-  const contactPointsState = useGetContactPointsState(selectedAlertmanager ?? '');
+  const contactPointsState = useGetContactPointsState(
+    // Workaround to not try and call this API when we don't have access to the policies tab
+    showPoliciesTab && canSeeContactPointsStatus ? (selectedAlertmanager ?? '') : ''
+  );
 
   const {
-    currentData: result,
-    isLoading: resultLoading,
+    currentData,
+    isLoading,
     error: resultError,
-  } = useAlertmanagerConfig(selectedAlertmanager, {
-    refetchOnFocus: true,
-    refetchOnReconnect: true,
-  });
+    refetch: refetchNotificationPolicyRoute,
+  } = useNotificationPolicyRoute({ alertmanager: selectedAlertmanager ?? '' }, { skip: !showPoliciesTab });
```

Comment:
I see this contact point is not returning the auto-generated routes. Using k8s api admins cannot see this autogenerated node in the tree. Is this the expected behaviour?.

Reply:
We aren't going to return autogenerated policies with the new API (at least not at the start). The admin could still see the raw routes in the settings view though. With the contact points changes, users can see contact point usage in simplified routing, which I believe was the main motivation for showing autogenerated routes?

cc @JacobsonMT

Reply:
Discussed offline; let's revisit this later and it isn't dependent on additional changes right now